surefire-its/src/test/resources/junit-platform/src/test/java/junitplatform/DisplayNameTest.java

```diff
@@ -0,0 +1,37 @@
+package junitplatform_1_0_0;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.logging.Logger;
+
+class DisplayNameTest
+{
+    private static final Logger LOGGER = Logger.getLogger( DisplayNameTest.class.getName() );
+
+    @Test
+    @DisplayName( "Custom test name containing spaces" )
```

Comment:
Would it be useful to have some unit test around display name which contains '(' and ')' ?